torch/csrc/jit/python/script_init.cpp

```diff
@@ -471,13 +476,18 @@ static void setInputTensorTypes(Graph& g, const Stack& stack, bool complete) {
     if (auto named_type = v->type()->cast<c10::NamedType>()) {
       if (auto qualname = named_type->name()) {
         if (getCustomClass(qualname->qualifiedName())) {
-          s_iter++;
+          if (param_count_list.empty()) {
+            s_iter++;
+          } else {
+            s_iter += param_count_list[list_idx];
```

Comment:
potential list_idx out of range, maybe check for that instead in the condition above?

Reply:
Added condition check before the loop, then it will not out of range.
```cpp
  if (!param_count_list.empty()) {
    TORCH_INTERNAL_ASSERT(input_values.size() == param_count_list.size());
  }
```